src/routes/handles.route.ts

```diff
@@ -12,27 +12,12 @@ class HandlesRoute extends BaseRoute {
 
     private initializeRoutes() {
         this.router.get(`${this.path}`, this.injectRegistryMiddleware, this.handlesController.getAll);
+        this.router.post(`${this.path}/search`, this.injectRegistryMiddleware, this.handlesController.search);
```

Comment:
After thinking on this, I think `/list` is appropriate for this endpoint. I can explain more when we chat, but basically, you can either start from the full list of Handles (`/handles`) or you can start from a predefined list of Handles (`/handles/list`) and from there all of the current filter params work from the query string.